integrated_channels/sap_success_factors/models.py

```diff
@@ -113,6 +166,45 @@ def __repr__(self):
         """
         return self.__str__()
 
+    @property
+    def provider_id(self):
+        '''
+        Fetch ``provider_id`` from global configuration settings
+        '''
+        return SAPSuccessFactorsGlobalConfiguration.current().provider_id
+
+    def _payload_data(self, empty_value):
+        """
+        Convert the audit record's fields into SAP SuccessFactors key/value pairs.
+        """
+        return dict(
+            userID=self.sapsf_user_id,
+            courseID=self.course_id,
+            providerID=self.provider_id,
+            # SAP SuccessFactors requires strings, not boolean values.
+            courseCompleted="true" if self.course_completed else "false",
+            completedTimestamp=self.completed_timestamp,
+            grade=self.grade,
+            # TODO: determine these values from the enrollment seat?
+            price=empty_value,
+            currency=empty_value,
+            creditHours=empty_value,
+            # These fields currently have no edX source, so remain empty.
+            totalHours=empty_value,
+            contactHours=empty_value,
+            cpeHours=empty_value,
+            instructorName=empty_value,
+            comments=empty_value,
```

Comment:
@brittneyexline we should confirm with @griffresch on the empty fields we are sending here -- for example we may want to include some additional information in the 'comments' section.